### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by mydea**
This PR updates all the dependencies and applies the changes from `ember init`. Functionally, nothing changes, this is only internal.

### CODE REVIEW

File: tests/dummy/app/index.html [hunk 1]
```diff
@@ -3,23 +3,23 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>ember-toastr</title>
+    <title>Dummy</title>
```
Comment (knownasilya) [inline tests/dummy/app/index.html]: Title should probably be kept.

**Review by knownasilya — CHANGES_REQUESTED** [review]
Very small change, but otherwise 👍

File: tests/dummy/app/index.html (hunk 1 above)
Comment (mydea) [reply tests/dummy/app/index.html]: Ah, I knew I forgot something ;) I fixed this and pushed the change!

**Comment by knownasilya** [comment]
Published as v1.7.1

**Pull request merged**